lucille-core/src/main/java/com/kmwllc/lucille/util/CamelCaseConfigConverter.java

```diff
@@ -0,0 +1,114 @@
+package com.kmwllc.lucille.util;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.commons.text.CaseUtils;
+
+public class CamelCaseConfigConverter {
```

Comment:
We do need a unit test for this. Can be a simple test that runs the conversion on one input conf file and confirms that the output matches what we expect. The conf file should have some examples of edge cases like multiple instances of the same stage property name on one line (used as a property and as a value) and some other snake case properties in connectors or elsewhere that shouldn't be converted because they're not stage properties

Reply:
WIP

Reply:
created test for this in [1134f0a](https://github.com/kmwtechnology/lucille/pull/203/commits/1134f0ab1f7385d55758becbd4ca91cab1c4ba1f)